<?xml version="1.0"?>
<!-- This example uses predefined waypoint locations, entered as GPS locations. -->
<launch>

  <!-- Creates a Jackal in Gazebo. -->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="use_sim_time" value="true"/>
    <arg name="gui"          value="false"/>
    <arg name="world_name"   value="$(find multi_jackal_tutorials)/worlds/empty.world"/>
  </include>
  <!-- Jackal0 has GPS. -->
  <include file="$(find multi_jackal_base)/launch/jackal_base.launch">
    <arg name="ns"             value="jackal0"/>
    <arg name="x"              value="50.5"/>
    <arg name="y"              value="6.5"/>
    <arg name="use_global_tf"  value="false"/>
    <arg name="use_global_ekf" value="true"/>
  </include>

  <node ns="jackal0" pkg="gps_switch" type="gps_switch.py" name="gps_switch" output="screen">
    <param name="cutoff_time" value="10"/>
    <remap from="gps_in"  to="odometry/gps"/>
    <remap from="gps_out" to="odometry/gps_switched"/>
  </node>

</launch>
